src/__integrations__/pm.ts

```diff
@@ -29,6 +31,79 @@ walletRouter.post('/pp-restapi/v3/users/actions/start-session', function (req, r
   }
 });
 
+walletRouter.post('/pp-restapi/v3/wallet', function (req, res) {
+  WalletRequest.decode(req.body).fold(
+    () => res.sendStatus(500),
+    decodedReq => {
+      if (!req.headers.authorization?.match(/Bearer [\d\w]{128}/)) {
+        return res.sendStatus(401);
+      } else if (
+        // request id {}
+        Object.keys(decodedReq).length === 0
+      ) {
+        return res.sendStatus(500);
+      } else if (decodedReq.data && Object.keys(decodedReq.data).length === 0) {
+        return res.sendStatus(422);
+      } else if (decodedReq.data?.creditCard?.expireYear?.match(/(\d\d.+)|[^\d]+/)) {
+        return res.sendStatus(422);
+      } else if (!Object.prototype.hasOwnProperty.call(decodedReq.data, 'idPagamentoFromEC')) {
+        return res.sendStatus(500);
+      } else {
+        const obscuredPan = fromNullable(decodedReq.data?.creditCard?.pan as string)
+          .map(myPan => '*'.repeat(myPan?.length - 4) + myPan?.substr(myPan.length - 4))
+          .getOrElse('************4444');
+        return res.json({
+          data: {
+            idWallet: 40,
+            type: 'CREDIT_CARD',
+            favourite: false,
+            creditCard: {
+              id: 48,
+              holder: decodedReq.data?.creditCard?.holder,
+              pan: obscuredPan,
+              expireMonth: decodedReq.data?.creditCard?.expireMonth,
+              expireYear: decodedReq.data?.creditCard?.expireYear,
+              brandLogo: 'http://localhost:8080/wallet/assets/img/creditcard/generic.png',
+              flag3dsVerified: false,
+              brand: 'OTHER',
+              hpan: myFake.random.alphaNumeric(64),
+              onUs: false,
+            },
+            psp: {
+              id: 8,
+              idPsp: 'POSTE1',
```

Comment:
Could be useful to put all hardcoded data in to `mock` module file ?

Reply:
@pasqualespica I totally agree, but in my opinion we should avoid the standard  `__mock__` folder used by jest